src/app/shared/palette/collection/write-collection-field.component.ts

```diff
@@ -72,4 +74,27 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     return index ? `${this.caseField.label} ${displayIndex}` : this.caseField.label;
   }
 
+  openModal(i: number) {
+    const dialogConfig = new MatDialogConfig();
+    dialogConfig.disableClose = true;
+    dialogConfig.autoFocus = true;
+    dialogConfig.ariaLabel = 'Label';
+    dialogConfig.height = '220px';
```

Comment:
can this be moved to a css file?

Reply:
I tried, but since this is just a configurable object which takes these values.  So, unfortunately it is not possible put them in css file.